deploy/006-omnichain-executorOwner-remote.ts

```diff
@@ -86,25 +89,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const fastTrackTimelockAddress = (await ethers.getContract("FastTrackTimelock")).address;
   const criticalTimelockAddress = (await ethers.getContract("CriticalTimelock")).address;
 
-  const OmnichainGovernanceExecutor = await deploy("OmnichainGovernanceExecutor", {
-    from: deployer,
-    args: [LZ_ENDPOINTS[networkName], deployer],
-    log: true,
-    autoMine: true,
-  });
-
-  const omnichainGovernanceExecutor = (await ethers.getContractAt(
-    "OmnichainGovernanceExecutor",
-    OmnichainGovernanceExecutor.address,
-    ethers.provider.getSigner(deployer),
-  )) as OmnichainGovernanceExecutor;
-
+  const omnichainGovernanceExecutorAddress = (await ethers.getContract("OmnichainGovernanceExecutor")).address;
   const OmnichainExecutorOwner = await deploy("OmnichainExecutorOwner", {
     from: deployer,
-    args: [OmnichainGovernanceExecutor.address],
+    args: [omnichainGovernanceExecutorAddress],
     contract: "OmnichainExecutorOwner",
     proxy: {
-      owner: normalTimelockAddress,
+      owner: hre.network.live ? await getOmnichainGovernanceExecutorAdminAccount(networkName) : deployer,
```

Comment:
`getOmnichainGovernanceExecutorAdminAccount` accounts for hardhat

Reply:
It returns deployer's address.